test/shell-pipeline-tests/openvino-tensorflow-housing/pipelines-test-openvino-tensorflow-housing.sh

```diff
@@ -38,16 +38,16 @@ oc apply -k "$AIEDGE_E2E_PIPELINE_DIR_PATH"/test-data
 oc apply -k "$MANIFESTS_DIR"/
 
 ## prepare parameters
-AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH="$AIEDGE_E2E_PIPELINE_DIR_PATH"/example-pipelineruns/aiedge-e2e.pipelinerun-overridden.yaml
-cp "$AIEDGE_E2E_PIPELINE_DIR_PATH"/example-pipelineruns/aiedge-e2e.tensorflow-housing.pipelinerun.yaml "$AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH"
+AIEDGE_E2E_PIPELINE_OVERRIDDEN_PATH="$AIEDGE_E2E_PIPELINE_DIR_PATH"/example-pipelineruns/git-fetch.tensorflow-housing.pipelinerun-overridden.yaml
```

Comment:
Please also rename the variable name, otherwise it is very confusing.

```suggestion
GIT_FETCH_PIPELINE_OVERRIDDEN_PATH="$AIEDGE_E2E_PIPELINE_DIR_PATH"/example-pipelineruns/git-fetch.tensorflow-housing.pipelinerun-overridden.yaml
```